rest/admin_api.go

```diff
@@ -162,6 +162,12 @@ func (h *handler) handleCreateDB() error {
 		}
 	}
 
+	base.Audit(h.ctx(), base.AuditIDCreateDatabase,
+		base.AuditFields{
+			base.AuditFieldDatabase: dbName,
+			base.AuditFieldPayload:  rawBytes,
```

Comment:
Should we be performing redaction on the payload via `DbConfig.Redacted()` before logging it, or is that done elsewhere in the audit logging framework?

Reply:
Added a redaction that works on the string because of the possibility that we nil a field and this wouldn't show up in Redacted() and deserialization.

None of this code is performance sensitive because we are loading a new database but I'm thinking we might want a call in other audit events of `IsAuditLoggingEnabled` before performing new operations.